flank-scripts/src/main/kotlin/flank/scripts/shell/firebase/GenerateJavaClientCommand.kt

```diff
@@ -14,9 +14,10 @@ object GenerateJavaClientCommand : CliktCommand(name = "generateJavaClient", hel
     override fun run() {
         checkIfPipInstalled()
         installClientGeneratorIfNeeded()
-        val apiPath = Paths.get("test_api").toString()
+        val firebaseApiPath = Paths.get("", "firebase_apis").toString()
```

Comment:
There is any difference between paths from ```Paths.get("", "firebase_apis").toString()``` and ```Paths.get("firebase_apis").toString()```?

Reply:
no, there is not, fixed